src/docker_bin/wo_docker_run

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+set -e
+
+## Argument checks.
+for arg in "$@"; do
+    if [ "$arg" = "--detach" ] || [ "$arg" = "-d" ]; then
+        echo "Should not give '--detach' argument to $0" >&2
+        exit 1
+    fi
+    if [ "$arg" = "--rm" ]; then
+        echo "Should not give '--rm' argument to $0" >&2
+        exit 1
+    fi
+done
+if [ "$SLURM_JOB_ID" = "" ]; then
+    echo "Can not find SLURM_JOB_ID environment variable" >&2
+    exit 1
+fi
+
+## Start running the container in detach mode.
+## Also use --pid=host for protecting mental health.
+WO_CONTAINER_NAME=`docker run -d --pid host $@`
```

Comment:
We can't run the container detached, since it gets its script via STDIN. Instead, do
```
docker run ... /bin/bash - <<"DELIMITER" &
script goes here
...
DELIMITER
pid=$!
```

(this is the approach that we use in task.py in wolF).

Reply:
Now `wo_docker_run` redirects its stdin to `docker run`, which means something like 
```
wo_docker_run ... /bin/bash <<EOF
...
EOF
```
should work.